modules/vaos/app/services/eps/appointment_service.rb

```diff
@@ -12,5 +12,16 @@ def get_appointments
                          {}, headers)
       OpenStruct.new(response.body)
     end
+
+    ##
+    # Create draft appointment in EPS
+    #
+    # @return OpenStruct response from EPS create draft appointment endpoint
+    #
+    def create_draft_appointment(referral_id:)
+      response = perform(:post, "/#{config.base_path}/appointments",
+                         { patientId: patient_id, referralId: referral_id }, headers)
```

Comment:
Is referralId passed as a top level param like this correct? wellhive's POST /appointments api seems to be requiring it as a `referral` object: https://wellhive.github.io/api-docs/#/Appointment/Appointment%23create

Reply:
Not sure if the document is outdated.  Currently, the API calls in postman works with `referralId` passed as top-level parameter.

Reply:
ok great, yeah the api doc might be outdated. Can you also test it using rails console and make sure it works?

Reply:
Yeah, I verified calling this API in rails console & it works.